src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");
```

Comment:
Similar to the 'doWork' method, the stop message should also indicate the specific type of Machine. Include 'Bulldozer' in the message to improve its informativeness.